accepted/0000-sbom-command.md

```diff
@@ -0,0 +1,461 @@
+    },
+    {
+      "referenceCategory": "PACKAGE-MANAGER",
+      "referernceType": "purl",
+      "referenceLocator": "pkg:npm/debug@4.3.4"
+    }
+  ],
+  "checksums": [
+    {
+      "algorithm": "SHA512",
+      "checksumValue": "3d15851ee494dde0ed4093ef9cd63b..."
+    }
+  ]
+}
+```
+
+The example record shown above conforms to version 2.3 of the SPDX [Package](https://spdx.github.io/spdx-spec/v2.3/package-information/) specification.
+
+The <code>[downloadLocation](https://spdx.github.io/spdx-spec/v2.3/package-information/#77-package-download-location-field)</code> field will report the resolved location calculated by Arborist. This may point to a package registry, or a git URL for dependencies referenced directly from git. The top-level project will specify a value of <code>NOASSERTION</code> for the download location as this information is not available.
+
+All packages will specify a `false` value for the <code>[filesAnaylzed](https://spdx.github.io/spdx-spec/v2.3/package-information/#78-files-analyzed-field)</code> field to signal that no assertions are being made about the files contained within a package.
+
+The <code>[externalRefs](https://spdx.github.io/spdx-spec/v2.3/package-information/#721-external-reference-field)</code> field will contain two <code>[PACKAGE-MANAGER](https://spdx.github.io/spdx-spec/v2.3/external-repository-identifiers/#f3-package-manager)</code> references, one using the <code>[npm](https://spdx.github.io/spdx-spec/v2.3/external-repository-identifiers/#f32-npm)</code> reference type and the other using the <code>[purl](https://spdx.github.io/spdx-spec/v2.3/external-repository-identifiers/#f35-purl)</code> reference type.
+
+
+## Unresolved Questions and Bikeshedding
+
+* There’s a case to be made that SBOM generation should just be a flag on the existing `npm-ls` command – an SBOM is really just a different format of the output already generated by `npm-ls` (in fact, the CycloneDX tool described above parses the output from `npm-ls` in order to generate its SBOM). However, many of the arguments supported by `npm-ls` don’t really make sense in the context of SBOM generation (package spec filtering, depth, etc…) so it feels like a bit of an awkward fit. \
+ \
+Making it a distinct command allows us to add SBOM-specific features in the future like a `--sign` flag which could be used to generate a signed SBOM. \
+ \
+_Recommendation: Add a distinct command for generating an SBOM._
+ 
+* Does `npm-sbom` command have a notion of a “default” SBOM format? Do we give preference to one of CycloneDX/SPDX or do we remain totally neutral (possibly at the expense of DX)? \
+ \
+_Recommendation: Default to SPDX if no format is specified._
+
+* Both CycloneDX and SPDX support multiple document formats (JSON, XML, Protocol Buffers, etc). Should we support output of multiple formats, or do we stick w/ JSON? \
+ \
+_Recommendation: Stick with JSON-only for the first version of this feature._
```

Comment:
agree.
NPM is a package manager for the JavaScript-centered `nodeJS`. 
JavaScript's native data format (for serialization and information-interchange) is JSON.